src/client/components/Profile.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { PropTypes } from 'react'
+import userObject from '../../tests/client/fixtures/userObject';
+import ViewProfile from './ViewProfile';
+import Loading from './Loading';
+
+/* Profile Controller Component
+ * Controller for the Container Component ProfileHolder
+ * Handles requesting a given user id from the server
+ * and rendering a profile view based on that data.
+ * If the user is viewing their own profile, this component
+ * will render EditProfile, which contains logic to update user records.
+ * Otherwise, it will render ViewProfile, which contains logic for the currently
+ * Logged-in user to follow/unfollow/DM the user being viewed.
+ */
+
+function fetchUser(id) {
```

Comment:
I think the fetchUser and receiveViewUser functions could be parsed out into separate action creators and reducers to enable serialization of actions and clarify the status of the async process -- e.g. FETCH_USER_REQUEST vs. FETCH_USER_SUCCESS vs. FETCH_USER_FAILURE.